resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -5,10 +5,12 @@ restartfile=<%= EZBake::Config[:restart_file] %>
 timeout=<%= EZBake::Config[:start_timeout] %>
 
 if [ -r "$restartfile" ];  then
-    kill -HUP $(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
+    pid=$(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
+    kill -HUP $pid
     cur=$(head -n 1 "$restartfile")
     initial=$cur
     while [ "$cur" == "$initial" ] ;do
+        kill -0 $pid >/dev/null 2>&1
```

Comment:
Yeah, that's all what I've seen / am expecting.  I would expect logs to have other data about the failure.  I went back/forth on this a bit, too.  I'm open to echoing a specific error message out here if others think it would be valuable.  Probably would have to do a `set +e` and `set -e` around it so the error can be caught and message written.